src/ngx_http_lua_common.h

```diff
@@ -176,6 +178,9 @@ struct ngx_http_lua_main_conf_s {
 
     ngx_array_t         *preload_hooks; /* of ngx_http_lua_preload_hook_t */
 
+    ngx_array_t         *delay_list; /* of void* */
+    unsigned             delay_list_inited;
```

Comment:
@agentzh  `delay_list` is used for register all third part module which need to delay `init_by_lua`, `delay_list_inited` is used for count how many module has run `ngx_http_lua_delay_init_phase`, and used to determine whether `init_by_lua` should be executed